app/routes.py

```diff
@@ -1,2 +1,296 @@
+                "phone": customer.phone,
+                "videos_checked_out_count": customer.videos_checked_out_count
+            })
+        return jsonify(customers_response), 200
+    
+    # CREATES (POST) a new customer
+
+    elif request.method == "POST":
+
+        request_body = request.get_json()
+        if "name" not in request_body or "postal_code" not in request_body or "phone" not in request_body:
+            return jsonify({"details": "Invalid data"}), 400
+        else:
+            new_customer = Customer(
+                name=request_body["name"], 
+                postal_code=request_body["postal_code"], 
+                phone=request_body["phone"],
+                registered_at=datetime.now()
+                )
+
+            db.session.add(new_customer)
+            db.session.commit()
+
+            return new_customer.to_json(), 201
+        
+# endpoint retrieves, updates and deletes records of a specific customer
+@customers_bp.route("/<customer_id>", methods=["GET", "PUT", "DELETE"], strict_slashes=False)
+def specific_customer(customer_id):
+
+    # GET details about specific customer
+
+    customer = Customer.query.get(customer_id)
+
+    if customer is None:
+        return make_response(), 404
+    
+    elif request.method == "GET":
+        return customer.to_json(), 200
+
+    # UPDATE (PUT) & return details about specific customer
+
+    elif request.method == "PUT":
+        form_data = request.get_json()
+
+        if "name" not in form_data or "postal_code" not in form_data or "phone" not in form_data:
+            return jsonify({"details": "Invalid data"}), 400
+        
+        else:
+            customer.name = form_data["name"]
+            customer.postal_code = form_data["postal_code"]
+            customer.phone = form_data["phone"]
+
+            db.session.commit()
+            return customer.to_json(), 200
+    
+    # DELETE a specific customer
+    elif request.method == "DELETE":
+        db.session.delete(customer)
+        db.session.commit()
+        return jsonify({
+            "id": customer.id,
+            "details": f'Customer {customer.name} successfully deleted'}), 200
+
+
+# <------------------------------------- VIDEO ENDPOINTS ------------------------------------->
+
+videos_bp = Blueprint("videos", __name__, url_prefix= "/videos")
+
+# endpoint retrieves details about ALL videos and creates a record of a video
+@videos_bp.route("", methods=["GET", "POST"], strict_slashes=False)
+def video_details():
+
+    # GET all existing videos and details about each video
+    
+    if request.method == "GET":
+
+        all_videos = Video.query.all()
+
+        videos_response = []
+        for video in all_videos:
+            videos_response.append({
+                "id": video.id,
+                "title": video.title,
+                "release_date": video.release_date,
+                "total_inventory": video.total_inventory,
+                "available_inventory": 0
```

Comment:
This value should not be hardcoded to `0`, but rather read from the value stored in the instance of the model, or calculated as the `total_inventory` - the length of the rentals for that video.